tests/phpunit/mockers/class-wc-mocker-gateway-amazon-payments-advanced.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Amazon Pay Gateway Mocker Class.
+ *
+ * @package WC_Gateway_Amazon_Pay/Tests
+ */
+
+declare(strict_types=1);
+
+/**
+ * Amazon Pay Gateway Mocker Class.
+ */
+class WC_Mocker_Gateway_Amazon_Payments_Advanced extends WC_Gateway_Amazon_Payments_Advanced {
```

Comment:
Is our codebase allows us to use standard mocking APIs of PHPUnit, or do we have to spawn the mock classes?